src/modules/main/types/post.ts

```diff
@@ -0,0 +1,43 @@
+export interface IPost {
+    postId: number,
+    authorId: number,
+    directionId: number,
+    typeId: number,
+    title: string,
+    content: string,
+    status: PostStatus,
+    important: boolean,
+    tags: string,
+    createdAt: Date,
+    modifiedAt: Date
+}
```

Comment:
pls add "preview" property, which will be used as a preview

Reply:
ok, but what type it will be? 
what will we receive in this property?
thank you